src/screens/NewsScreen/Search.js

```diff
@@ -107,7 +104,7 @@ const styles = StyleSheet.create({
     borderBottomWidth: 1,
     borderBottomColor: COLORS.alto,
     height: 50
-  }),
+  })
 });
 
-export default React.memo (Search);
+export default React.memo(Search);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There's a minor issue with the formatting of the export statement. It's best to avoid unnecessary spaces between function names and parentheses.
```diff
- export default React.memo (Search);
+ export default React.memo(Search);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->